lib/types/CredentialIssuance.types.ts

```diff
@@ -57,7 +55,30 @@ export interface JWK {
   n?: string;
 }
 
-export type Alg = 'ES256' | 'EdDSA';
+export interface ProofOfPossessionCallbackArgs {
+  privateKey: unknown;
+  publicKey: unknown;
+  header: JWTHeader;
+  payload: JWTPayload;
+  [x: string]: unknown;
+}
+
+export interface ProofOfPossessionOpts {
+  clientId?: string;
+  issuerURL?: string;
+  proofOfPossessionCallback: JWTSignerCallback;
+  proofOfPossessionVerifierCallback?: JWTVerifyCallback;
+  proofOfPossessionCallbackArgs: ProofOfPossessionCallbackArgs;
```

Comment:
Why aren't these anr argument of the actual signer callback?

Reply:
added `clientId` and `issuerURL` to `ProofOfPossessionCallbackArgs` properties

Reply:
Point is that I would not expect the CallbackArgs in the interface to begin with. The callback function you have to implement gets the CallbackArgs passed in. So the implementation needs to be able to accept these arguments. Why would we list these in the ProofOfPossesionArgs? The values are mostly created and supplied to the callback